CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing to Tlog
+The Tlog project welcomes new contributors. This document will guide you through the process.
+
+## Pull Requests
+We actively welcome your pull requests. There are a few guidelines that we need
+contributors to follow so that we can try to keep the codebase consistent and clean.
+
+* Fork the main repository and clone it to your local machine.
+* Create a topic branch from master. Please avoid working directly on the ```master``` branch.
+* Make commits of logical units, and make sure to follow the CODINGSTYLE.
+* Submit the Pull Request(PR) on Github.
+
+## Issues
+We use GitHub [issues tracker](https://github.com/Scribery/tlog/issues) to track public bugs.
+Please ensure your description is clear and has sufficient instructions to be able to reproduce the issue.
+
+## Coding Style guidelines
+
+* Spaces are used for indentation, should not use tabs.
+
+* Lines should be at most 78 chars.
+
+* Do not leave trailing whitespaces at the end of line.
+
+* Braces open on the same line as the for/while/if/else/etc. Closing
+  braces are put on a line of their own, except in the else of an if statement
+  or in a while of a do-while statement. Always use braces for if and while.
+
+```c
+if (a == b) {
+  	...
+}
+
+if (a == b) {
+    ...
+    ...
+} else if (a > b) {
+    ...
+    ...
+}
+
+if (a == b) {
+  	...
+} else {
+  	do_something_else ();
+}
```

Comment:
Please observe indenting guidelines here.

Reply:
Indenting is fine here now, but please don't put a space before the opening parentheses in a function call.